nipype/interfaces/freesurfer/preprocess.py

```diff
@@ -1528,7 +1528,7 @@ class MNIBiasCorrection(FSCommand):
     output_spec = MNIBiasCorrectionOutputSpec
 
     def _list_outputs(self):
-        outputs = self._outputs().get()
+        outputs = self._output_spec().get()
```

Comment:
Since most other interfaces are using outputspec - I wanted to test. I can revert this